app/_src/gateway-operator/customization/pod-template-spec.md

```diff
@@ -2,10 +2,25 @@
 title: Using PodTemplateSpec
 ---
 
-The `DataPlane` resource uses the Kubernetes [PodTemplateSpec](/gateway-operator/{{ page.release }}/customization/pod-template-spec/) to define how pods should run. You can customize everything you need to in your deployment using this resource.
+The Kubernetes `PodTemplateSpec` defines how pods should run. You can customize everything you need to in your deployment using this resource.
 
 For more information see the [PodTemplateSpec documentation](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-template-v1/#PodTemplateSpec).
 
+A `PodTemplateSpec` can be provided on both `GatewayConfiguration` and `DataPlane` resources.
+
+## GatewayConfiguration
+
+When using the `GatewayConfiguration` resource, the `ControlPlane` and `DataPlane` resources are configured independently.
+
+* `ControlPlane` pods take their configuration from the `spec.controlPlaneOptions.deployment.podTemplateSpec` field.
+* `DataPlane` pods take their configuration from the `spec.dataPlaneOptions.deployment.podTemplateSpec` field.
```

Comment:
There are more options to configure those 2 CRDs:

- https://docs.konghq.com/gateway-operator/latest/reference/custom-resources/#controlplane
- https://docs.konghq.com/gateway-operator/latest/reference/custom-resources/#dataplane

respectively.

What we're trying to say here is that `spec.controlPlaneOptions.deployment.podTemplateSpec` and `spec.dataPlaneOptions.deployment.podTemplateSpec` can configure the Pods that will run Kong and KIC.

I might be biased but rewording this to make sure this is clear might be worth doing.

Reply:
Let's leave this as it and merge what's already in this PR 👍